apps/backend/src/libs/modules/plugins/libs/constants/open-routes.ts

```diff
@@ -0,0 +1,6 @@
+const OPEN_ROUTES: Set<RegExp> = new Set([
```

Comment:
The common name for this is white routes
```suggestion
const WHITE_ROUTES: Set<RegExp> = new Set([
```